src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,43 @@
+.nav {
+  &__list {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: inline-block;
+    padding: 23px 0;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $color-main-accent;
+    font-weight: 500;
+    line-height: 14px;
+  }
+
+  &__link:hover {
+    color: $color-blue-accent;
+  }
+}
+
+.is-active {
```

Comment:
in a perfect world `is-active` should be in a separate file, but in this case, you can leave it here